lib/learn_test/strategies/pytest/requirements_checker.rb

```diff
@@ -0,0 +1,75 @@
+require 'open3'
+
+module LearnTest
+  module Pytest
+    class RequirementsChecker
+      def self.check_installation
+        new.check_installation
+      end
```

Comment:
Could we simplify the classes in this file by putting the logic in the class methods directly instead of doing this class method calling an instance method dance? Or is there some metaprogram-y wizardry afoot?

Reply:
good question -- to tell the truth I didn't even look closely and was following the existing pattern. I will remove - don't see any value added there, especially because that instance poofs immediately.